packages/core/src/modules/connections/ConnectionsModule.ts

```diff
@@ -181,8 +181,8 @@ export class ConnectionsModule {
     return connectionRecord
   }
 
-  public async returnWhenIsConnected(connectionId: string): Promise<ConnectionRecord> {
-    return this.connectionService.returnWhenIsConnected(connectionId)
+  public async returnWhenIsConnected(connectionId: string, options?: { timeOutMs: number }): Promise<ConnectionRecord> {
```

Comment:
In general timeout is written in lowercase

```suggestion
  public async returnWhenIsConnected(connectionId: string, options?: { timeoutMs: number }): Promise<ConnectionRecord> {
```